pkgs/desktops/lxde/core/lxsession/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib
+, stdenv
+, fetchgit
+, autoconf
+, automake
+, docbook_xml_dtd_412
+, docbook_xsl
+, intltool
+, libxml2
+, libxslt
+, pkg-config
+, wrapGAppsHook
+, gtk2-x11
+, libX11
+, polkit
+, vala
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lxsession";
+  version = "0.5.5";
+
+  src = fetchgit {
+    url = "git://git.lxde.org/git/lxde/lxsession";
+    rev = version;
+    sha256 = "17sqsx57ymrimm5jfmcyrp7b0nzi41bcvpxsqckmwbhl19g6c17d";
+  };
+
+  patches = [ ./xmlcatalog_patch.patch ];
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    docbook_xml_dtd_412
+    docbook_xsl
+    intltool
+    libxml2
+    libxslt
+    pkg-config
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    gtk2-x11
+    libX11
+    polkit
+    vala
+  ];
+  
+  configureFlags = [
+    "--enable-man"
+    "--disable-buildin-clipboard"
+    "--disable-buildin-polkit"
+    "--with-xml-catalog=${docbook_xml_dtd_412}/xml/dtd/docbook/catalog.xml"
+  ];
+
+  preConfigure = "./autogen.sh";
+
+  meta = with lib; {
+    description = "Classic LXDE session manager";
+    license = licenses.gpl2;
```

Comment:
`licenses.gpl2` is deprecated, see https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206.

Reply:
Changed to gpl2Plus according to https://src.fedoraproject.org/rpms/lxsession/blob/master/f/lxsession.spec#L25.